### PR TITLE
external/stdk: update wifi wrapper on esp32

### PR DESCRIPTION
**Pull request by SRCN-LiQin**
update stdk wifi wrapper base on new wifi_manager refinement
1. modify kconfig name in defconfig
2. get wifi info with different api on esp32, wifi manager api
rely on ip address set in netlib, which is not applied on esp32

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [d5231c6855a3dbe0a21ab55b1454858de9f7e50b] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [d5231c6855a3dbe0a21ab55b1454858de9f7e50b] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [c034b3fea35d146e7d67d13c2f41ca1fb3617960] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [c034b3fea35d146e7d67d13c2f41ca1fb3617960] - Code Rule Check (C++) OK.

File: external/stdk/tizenrt-wrapper/os/iot_os_util_tizenrt.c [hunk 1]
```diff
@@ -84,8 +86,8 @@ void iot_os_thread_delete(iot_os_thread thread_handle)
 		return;
 	}
 
-	pthread_t pid = (pthread_t)thread_handle;
-	pthread_cancel(pid);
+	pthread_t *pid_h = (pthread_t *)thread_handle;
+	pthread_cancel(*pid_h);
```
Comment (zhouxinhe) [inline external/stdk/tizenrt-wrapper/os/iot_os_util_tizenrt.c]: free(pid_h) is needed.

**Comment by seinfra** [comment]
Target : [98f71f1b7c60e81397df86583b16c039273cc98f] - Code Rule Check (C++) OK.

File: external/stdk/tizenrt-wrapper/os/iot_os_util_tizenrt.c (hunk 1 above)
Comment (SRCN-LiQin) [reply external/stdk/tizenrt-wrapper/os/iot_os_util_tizenrt.c]: fixed, thanks

**Comment by seinfra** [comment]
Target : [98f71f1b7c60e81397df86583b16c039273cc98f] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [aafe99fccf8c0b104c76dd993719494ef0bb4372] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [aafe99fccf8c0b104c76dd993719494ef0bb4372] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [654d7ae434019bf027bceedbb347fe609877ff1c] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [654d7ae434019bf027bceedbb347fe609877ff1c] - Code Rule Check (C++) OK.

**Pull request merged**